src/style.css

```diff
@@ -1,3 +1,77 @@
+/* stylelint-disable font-family-name-quotes */
+
+:root {
+  --main-blue: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: 'Roboto', sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, you could specify a generic font family as a fallback: `font-family: 'Roboto', 'Arial', sans-serif;`.